public_html/composer.json

```diff
@@ -0,0 +1,129 @@
+{
```

Comment:
-[ ] SHOULD - If we do this, it seems like we'd want to remove .docker/install-plugins-themes.sh

-[ ] SHOULD - The docker readme should be updated to reflect ^ and the need to run composer install during provision, and composer update periodically

Reply:
Done in 2e4a7122, except updating composer programatically. I'll open a separate issue for doing this, along with pulling the latest to the core branch.